uaclient/tests/test_cli_attach_premium.py

```diff
@@ -0,0 +1,111 @@
+import mock
+
+from uaclient.testing.fakes import FakeConfig
+
+import pytest
+
+from uaclient.cli import (
+    action_attach_premium,
+    attach_premium_parser,
+    get_parser,
+)
+from uaclient.exceptions import AlreadyAttachedError, NonRootUserError
+
+M_PATH = "uaclient.cli."
+
+BASIC_MACHINE_TOKEN = {
```

Comment:
This looks familiar, is there a way we can reuse an existing definition of a basic machine token?

Reply:
Looking at this now.

Reply:
Fix pushed.